projects/ontimize-web-ngx/src/lib/components/form/navigation/o-form.navigation.class.ts

```diff
@@ -166,7 +166,7 @@ export class OFormNavigationClass {
     }
     this.cacheStateSubscription = formCache.onCacheStateChanges.subscribe(() => {
       const initialStateChanged = this.form.isInitialStateChanged();
-      const triggerExitConfirm = this.form.isInitialStateChanged(this.form.ignoreOnExit);
+      const triggerExitConfirm = this.form.confirmExit ? this.form.isInitialStateChanged(this.form.ignoreOnExit) : this.form.confirmExit;
```

Comment:
maybe you could change it for this.form.confirmExit && this.form.isInitialStateChanged(this.form.ignoreOnExit)